Rock/Communication/Transport/Firebase.cs

```diff
@@ -0,0 +1,244 @@
+﻿// <copyright>
+// Copyright by the Spark Development Network
+//
+// Licensed under the Rock Community License (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.rockrms.com/license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Net.Mail;
+
+using Rock.Attribute;
+using Rock.Data;
+using Rock.Model;
+using Rock.Web.Cache;
+using FCM.Net;
+using System.Threading.Tasks;
+
+namespace Rock.Communication.Transport
+{
+    /// <summary>
+    /// Communication transport for sending push notifications using Firebase
+    /// </summary>
+    [Description( "Sends a communication through Firebase API" )]
+    [Export( typeof( TransportComponent ) )]
+    [ExportMetadata( "ComponentName", "Firebase" )]
+    [TextField( "SenderId", "Your Firebase account sender Id (find at https://console.firebase.google.com/project/<YOUR_APP>/settings/cloudmessaging)", true, "", "", 0 )]
```

Comment:
I removed SenderId. No sense in having something there that isn't being used. Great catch! :100: